staging/src/k8s.io/apiserver/pkg/util/flowcontrol/apf_controller.go

```diff
@@ -221,10 +255,14 @@ func (cfgCtl *configController) initializeConfigController(informerFactory kubei
 		}})
 }
 
+func (cfgCtl *configController) WaitForCacheSync(stopCh <-chan struct{}) bool {
+	return cache.WaitForCacheSync(stopCh, cfgCtl.plInformerSynced, cfgCtl.fsInformerSynced)
```

Comment:
nit: we don't normally have functions for one liners.

Reply:
This is a public method of `TestableInterface`.